### DISCUSSION TITLE
AXE5-Eagle User Guide Table of Contents

### DISCUSSION THREAD

**Discussion by ghost** [ArrowElectronics/Agilex-5#3] [Ideas]
Hi, will it be possible to generate the AXE5-Eagle User Guide with bookmarks/active table of contents? This would make navigating the document much more efficient. Thank you.

In Word it can be done as follows: File > Export > Create PDF/XPS > Options > 'follow image instructions' > Publish
![image](https://github.com/user-attachments/assets/e873e853-0262-4be9-bb18-dc8ea77fe440)

**Comment by skravats** [comment]
Hi NicWre

Thanks for the suggestion. The document has been updated to include bookmarks.
